.gitignore

```diff
@@ -8,4 +8,4 @@ target
 /build
 /*.iml
 /*.ipr
-/*.iws
+/*.iws
```

Comment:
That's just minor, but you should agree with me that it is jut redundant noisy change.
The file has had a new line in the end before, but your IDE destroyed that and added the extra work for us :smile: 

As I said: minor. And will be fixed on merge. But just for the case you'd like to save some your time against my similar comments in the future.